devenv/checks/credsStore.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import json
+import os
+import shutil
+
+from devenv.lib_check.types import checker
+from devenv.lib_check.types import fixer
+
+tags: set[str] = {"builtin"}
+name = "credsStore"
```

Comment:
unused? also mypy wouldn't require this annotation since it can infer

Reply:
it's used during construction of `doctor.Check`s